### DISCUSSION TITLE
@macmarrum: bug in your script "swap places with parent"

### DISCUSSION THREAD

**Discussion by autoca** [freeplane/freeplane#693] [Help]
Hi macmarrum,

Your script "swap places with parent" seems to have a bug when trying to swap places with a parent that has a picture / image inserted.
I got the feeling that such a picture may also affect other nodes in that branch to not work with your script anymore, but I could not establish a pattern for that extended effect.

At least, I could isolate it for the simple scenario: parent with pic -> child without pic.
Please, see attached sample mindmap.

[test with pic.zip](https://github.com/freeplane/freeplane/files/9656243/test.with.pic.zip)

MTIA and regards, a.

**Comment by macmarrum** [comment]
Let's continue in https://github.com/macmarrum/freeplane-scripting/issues/1
